<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customer</class>
 <widget class="QWidget" name="customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget {
    background-color: #F5E3CB;
}
</string>
  </property>
  <widget class="QLabel" name="labelResultCustomer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewCustomer">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>331</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
    background-color: transparent;
    gridline-color: #b5b5b5; 
}

QTableView::item {
    border: 1px solid #b5b5b5; 
}

QHeaderView::section {
    background-color: transparent;
}
</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>511</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonAddCustomer">
      <property name="styleSheet">
       <string notr="true">QPushButton {

font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: #344B61;
    border: 2px solid #F57B56; 
    border-radius: 5px;

}

QPushButton:hover {
    background-color: #344B61; 
    color: #ffffff; 
}


</string>
      </property>
      <property name="text">
       <string>Add Customer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonEditCustomer">
      <property name="styleSheet">
       <string notr="true">QPushButton {

font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: #344B61;
    border: 2px solid #F57B56; 
    border-radius: 5px;

}

QPushButton:hover {
    background-color: #344B61; 
    color: #ffffff; 
}


</string>
      </property>
      <property name="text">
       <string>Edit Customer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonRemoveCustomer">
      <property name="styleSheet">
       <string notr="true">QPushButton {

font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: #344B61;
    border: 2px solid #F57B56; 
    border-radius: 5px;

}

QPushButton:hover {
    background-color: #344B61; 
    color: #ffffff; 
}


</string>
      </property>
      <property name="text">
       <string>Remove Customer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>125</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelCustomerId">
      <property name="styleSheet">
       <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
      </property>
      <property name="text">
       <string>Id:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelCustomerName">
      <property name="styleSheet">
       <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
      </property>
      <property name="text">
       <string>Name: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelCustomerLastName">
      <property name="styleSheet">
       <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
      </property>
      <property name="text">
       <string>Last Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelPhoneNumber">
      <property name="styleSheet">
       <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
      </property>
      <property name="text">
       <string>Phone Number: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelEmail">
      <property name="styleSheet">
       <string notr="true">font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: rgb(50, 50, 50);
</string>
      </property>
      <property name="text">
       <string>E-mail: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>263</y>
     <width>135</width>
     <height>156</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEditCustomerId">
      <property name="styleSheet">
       <string notr="true">QLineEdit { background-color: transparent; }
</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditCustomerName">
      <property name="styleSheet">
       <string notr="true">QLineEdit { background-color: transparent; }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditCustomerLastName">
      <property name="styleSheet">
       <string notr="true">QLineEdit { background-color: transparent; }</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPhone">
      <property name="styleSheet">
       <string notr="true">QLineEdit { background-color: transparent; }</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditEmail">
      <property name="styleSheet">
       <string notr="true">QLineEdit { background-color: transparent; }</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
